src/calculateRentalCost.js

```diff
@@ -3,8 +3,22 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const MEDIUM_TERM_DUSCOUNT = 20;
```

Comment:
There is a typographical error in the variable name `MEDIUM_TERM_DUSCOUNT`. It should be `MEDIUM_TERM_DISCOUNT` to correctly spell 'discount'.